### PR TITLE
Adding NavBar with BootStrap

### PR DESCRIPTION
**Pull request by sarahz916**
Added Navigation Bar across the website for easier navigation. 
Bug: Navigation Bar collapses but does not expand as expected when viewed on smaller screens.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>loas-zous</deploy.projectId>
```
Comment (JKubs15) [inline portfolio/pom.xml]: I'm not sure if this is the correct project ID. I think it should be 'zous-step-2020'.

File: portfolio/src/main/webapp/contact.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sarah Zou's STEP  Portfolio</title>
```
Comment (JKubs15) [inline portfolio/src/main/webapp/contact.html]: Is there a reason there are 2 spaces between STEP and Portfolio?

**Review by JKubs15 — APPROVED** [review]

File: portfolio/pom.xml (hunk 1 above)
Comment (kkincy) [reply portfolio/pom.xml]: Right, it should be "zous-step-2020". I just set up the project here:

https://pantheon.corp.google.com/home/dashboard?project=zous-step-2020

File: portfolio/src/main/webapp/intro.html [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sarah Zou's STEP  Portfolio</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <!--<link rel="stylesheet" href="style.css">-->
+    <script src="script.js"></script>
+  </head>
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <a class="navbar-brand" href="#">Sarah Zou</a> <!-- NEED TO FIX COLLAPSE NAV BAR-->
```
Comment (kkincy) [inline portfolio/src/main/webapp/intro.html:11]: Is the navbar currently broken? Or is this something that needs to be tested / implemented later?

**Review by kkincy — APPROVED** [review]

**Pull request merged**